mipengine/node/tasks/remote_tables.py

```diff
@@ -6,6 +6,9 @@
 from mipengine.node.monetdb_interface import remote_tables
 
 
+# TODO: the database name has to come from somwhere outside this module. Not from the controller, controller should not know that
```

Comment:
@jassak @apmariglis  What exactly did we decide on this one?

The db_name will exist on the config file?

We need to create a ticket for this as well.

Reply:
Yes, the DB name is completely unrelated to the business logic and should only appear in the DB interface. Even if in the future we have many DBs (although I don't see why since we can have multiple tables in a single DB) it's not something the controller should know.

For now let's put it in the config file, but after the migration to toml pr is merged (very soon).